compiler/damlc/daml-doc/test/DA/Daml/Doc/Render/Tests.hs

```diff
@@ -299,13 +299,13 @@ mkExpectMD anchor name descr templates classes adts fcts
       , ""]
   ]
 
-renderTest :: RenderFormat -> (String, ModuleDoc) -> T.Text -> Tasty.TestTree
-renderTest format (name, input) expected =
+renderTest :: RenderFormat -> Map.Map Anchor String -> (String, ModuleDoc) -> T.Text -> Tasty.TestTree
```

Comment:
`Map.Map Anchor String` is used so often, I wonder if it would be better to use a type alias, or a newtype?

Also, it might be a good idea to switch to a `HashMap`.

Reply:
(Actually a newtype sounds good here, because the external anchor table needs to get passed around a lot as an opaque piece of data, and its representation is only needed in a few places.)

Reply:
Made it a newtype. Stopped short of changing it to a HashMap for now (did leave a TODO there).

Reply:
Now it's a HashMap 😄